docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -13,11 +13,21 @@ On this page, you can learn about the following topics:
 
 ## Incremental compilation
 
-The Kotlin Gradle plugin supports incremental compilation. Incremental compilation tracks changes to files in the classpath
-between builds so that only the files affected by these changes are compiled. Incremental compilation works with [Gradle's
-build cache](#gradle-build-cache-support) and supports [compilation avoidance](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_compile_avoidance).
-
-Incremental compilation is supported for Kotlin/JVM and Kotlin/JS projects, and is enabled by default.
+The Kotlin Gradle plugin supports incremental compilation, which is enabled by default for Kotlin/JVM and Kotlin/JS projects.
+Incremental compilation tracks changes to files in the classpath between builds so that only the files affected
+by these changes are compiled.
```

Comment:
Would saying "by the last changes" be accurate? If it is, I think it's clearer.

Reply:
I think it would be slightly inaccurate, if I understand correctly. Incremental compilation tracks all changes across the classpath, not just the most recent/last one.